diff --git a/node_modules/instagram-user/index.js b/node_modules/instagram-user/index.js
index 0360c4f..6287d90 100644
--- a/node_modules/instagram-user/index.js
+++ b/node_modules/instagram-user/index.js
@@ -1,6 +1,6 @@
 'use strict';
 const getEmails = require('get-emails');
-const got = require('got');
+const axios = require('axios');
 
 module.exports = async username => {
 	if (typeof username !== 'string') {
@@ -8,8 +8,17 @@ module.exports = async username => {
 	}
 
 	try {
-		const url = `https://instagram.com/${username}`;
-		const {graphql: {user}} = await got(url, {searchParams: {__a: 1}}).json();
+		const url = `https://instagram.com/${username}/?__a=1`;
+		const user = await axios({url: url, method: 'GET' })
+		.then(response =>{
+			//console.log('onfernoooo', response.data.graphql.user)
+			return response.data.graphql.user
+		//	let teste = JSON.parse(res.body)
+		//	return teste.graphql.user
+		}).catch(error => {
+			console.error('esse erro eh novo', error)
+		})
+		console.log('xxxxxx', user)
 		const email = getEmails(user.biography).values().next().value || '';
 
 		return {
@@ -25,10 +34,7 @@ module.exports = async username => {
 			website: user.external_url
 		};
 	} catch (error) {
-		if (error.response.statusCode === 404) {
-			error.message = `User "${username}" not found`;
-		}
-
+		console.error(error)
 		throw error;
 	}
 };
