diff --git a/node_modules/instory/index.js b/node_modules/instory/index.js
index c7f9f61..e40decc 100644
--- a/node_modules/instory/index.js
+++ b/node_modules/instory/index.js
@@ -2,7 +2,7 @@
 
 const got = require('got');
 
-module.exports = (username, exclude) => {
+module.exports = async (username, exclude) => {
 	const url = `https://api.storiesig.com/stories/${username}`;
 
 	const obx = {
@@ -12,13 +12,20 @@ module.exports = (username, exclude) => {
 
 	return got(url, {json: true}).then(res => {
 		const base = res.body.items;
-
-		const stories = {story: []};
-
+		let stories = [];
 		for (let i = 0; i < base.length; i++) {
-			base[i].video_versions === undefined ? stories.story.push(base[i].image_versions2.candidates[0].url) : stories.story.push(base[i].video_versions[0].url);
+			base[i].video_versions === undefined ? 
+			stories.push([{
+				"url": base[i].image_versions2.candidates[0].url, 
+				"shortcode": base[i].code,
+				"expiring_at": base[i].expiring_at
+				}]) : 
+			stories.push([{
+				"url": base[i].video_versions[0].url, 
+				"shortcode": base[i].code,
+				"expiring_at": base[i].expiring_at
+				}]);
 		}
-
 		return exclude !== undefined ? stories.story.filter(urls => urls.indexOf(obx[exclude]) === -1) : stories;
 	});
 };
\ No newline at end of file
